firefox_puppeteer/ui/browser_window.py

```diff
@@ -56,6 +56,14 @@ def tabbar(self):
         See the :class:`~ui.tabbar.Tabs` reference.
         """
 
+    @use_class_as_property('ui.autocompleteresults.AutocompleteResults')
+    def autocompleteresults(self):
+        """Provides utility members for accessing and manipulationg the
+        locationbar.
+
+        See the :class:`~ui.locationbar.LocationBar` reference.
+        """
+
```

Comment:
The autocomplete results are not an element related to the browser, but for the location bar which is also the parent. So please move this to the LocationBar class.